asr1k_neutron_l3/plugins/l3/schedulers/simple_asr1k_scheduler.py

```diff
@@ -47,17 +49,29 @@ def _get_candidates(self, plugin, context, sync_router):
 
             orig_candidates = plugin.get_l3_agents(context, active=True)
 
+            enabled_candidates = []
+            disabled_hosts = []
+            for candidate in orig_candidates:
+                if not jsonutils.loads(candidate.configurations).get('scheduling_disabled', False):
+                    enabled_candidates.append(candidate)
+                else:
+                    disabled_hosts.append(candidate.host)
+
+            if disabled_hosts:
+                LOG.debug(f'Ignoring agent hosts {", ".join(disabled_hosts)} for scheduling of {sync_router["id"]}, '
+                          'scheduling disabled')
```

Comment:
I'd use format strings here. The formatting is only done when the log level is low enough for it to print. It's good practice also for `error`, as I think sentry can use this to better aggregate on it. I'm not a 100% sure on it, though.